counterpartylib/lib/messages/issuance.py

```diff
@@ -124,6 +137,36 @@ def validate (db, source, destination, asset, quantity, divisible, callable_, ca
         if destination:
             problems.append('cannot transfer a non‐existent asset')
 
+    # validate parent ownership for subasset
```

Comment:
can you add a block here that's like:
```
if reissued_asset_longname and reissued_asset_longname != subasset_longname:
    problems.append('previous longname does not match longname specified in description')
```

just a good sanity check, I think (unless I'm missing something)